ecl/resource_ecl_provider_connectivity_tenant_connection_v2.go

```diff
@@ -0,0 +1,461 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/ecl/provider_connectivity/v2/tenant_connections"
+)
+
+func resourceProviderConnectivityTenantConnectionV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceProviderConnectivityTenantConnectionV2Create,
+		Read:   resourceProviderConnectivityTenantConnectionV2Read,
+		Update: resourceProviderConnectivityTenantConnectionV2Update,
+		Delete: resourceProviderConnectivityTenantConnectionV2Delete,
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"tenant_connection_request_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
```suggestion
				Required: true,
				ForceNew: true,
```

Reply:
Related to this.
https://github.com/nttcom/terraform-provider-ecl/pull/46#discussion_r375687948